api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -138,7 +138,7 @@
     Timestamp now = new Timestamp(clock.instant().toEpochMilli());
     workspaceService.getWorkspaceEnforceAccessLevelAndSetCdrVersion(
         workspaceNamespace, workspaceId, WorkspaceAccessLevel.WRITER);
-    long wId = workspaceService.get(workspaceNamespace, workspaceId).getWorkspaceId();
+    long dbWorkspaceId = workspaceService.get(workspaceNamespace, workspaceId).getWorkspaceId();
```

Comment:
nit: I like to mark everything final except when you definitely need to reassign it. Reasonable folks have disagreed in the case of locals though, since `final` is such a long modifier.